docs/import-data.rst

```diff
@@ -4,31 +4,44 @@
 Import data
 ===========
 
-This guide demonstrates how to import data into your CrateDB Cloud
-cluster. This how-to assumes that you already have a cluster running. If you
-need help starting a cluster, please see :ref:`tutorial on how to deploy a
-cluster for the first time <cloud-tutorials:cluster-deployment>`.
+This guide shows two methods of importing data into your CrateDB Cloud
+cluster. One is available directly in your Cloud Console, and the other in the
+Admin UI of your cluster.
 
 .. rubric:: Table of contents
 
 .. contents::
    :local:
 
+Import data from URL
+--------------------
 
-.. _AVC-d:
+The more convenient method for importing data is to import from a URL
+directly in your Cloud Console. See details `here`_.
+
+Import data in the Admin UI
+---------------------------
+
+If your data isn't publicly available or is in a format unsupported by URL
```

Comment:
maybe this would be clearer if we said something like

```
If your data isn't available on a public URL or is not in a format that is currently supported by the URL Import feature in the cloud console, you can import the data manually from the Admin 
UI.

```

Reply:
To be fair I would omit this altogether, as it is not true. You still need data to be publicly available somewhere to use `COPY FROM`. 

Maybe something like:
If you prefer, you can use a `COPY FROM` statement to import data directly into your CrateDB Cloud cluster.

Reply:
Ah, right. My bad